### PR TITLE
Minor tweaks to institution names

### PR DESCRIPTION
**Pull request by edwardhorsford**

### CODE REVIEW

**Review by frankieroberto — APPROVED** [review]

File: lib/dfe/reference_data/degrees/institutions.rb [hunk 1]
```diff
@@ -2376,17 +2388,20 @@ module Degrees
...
             suggestion_synonyms: [],
-            match_synonyms: ['University Academy 92'],
+            match_synonyms: ['UA92'],
             hesa_itt_code: '434',
             dttp_id: '203f182c-1425-ec11-b6e6-000d3adf095a',
             ukprn: '10067648',
             'has_never_awarded_degrees' => true },
           'e53e182c-1425-ec11-b6e6-000d3adf095a' =>
-          { name: 'TEDI-London',
+          { name: 'The Engineering and Design Institute London (TEDI-London)',
```
Comment (frankieroberto) [inline lib/dfe/reference_data/degrees/institutions.rb:2399]: The "TEDI-London" brand seems slightly more prominent than the full name "The Engineering and Design Institute London" on [their website](https://tedi-london.ac.uk) so I wonder if it's worth reversing this order, on the assumption that users are more likely to type in the acronym?

```suggestion
          { name: 'TEDI-London (The Engineering and Design Institute London)',
```

File: lib/dfe/reference_data/degrees/institutions.rb [hunk 2]
```diff
@@ -2376,17 +2388,20 @@ module Degrees
             ukprn: '10020381',
             'has_never_awarded_degrees' => true },
           '203f182c-1425-ec11-b6e6-000d3adf095a' =>
-          { name: 'UA92',
+          { name: 'University Academy 92',
```
Comment (frankieroberto) [inline lib/dfe/reference_data/degrees/institutions.rb:2391]: Any reason for switching to the long name? Looks like their main brand is UA92? https://ua92.ac.uk

**Review by frankieroberto — APPROVED** [review]

File: lib/dfe/reference_data/degrees/institutions.rb [hunk 3]
```diff
@@ -1932,7 +1932,7 @@ module Degrees
             ukprn: '10023434',
             'has_never_awarded_degrees' => true },
           '003e182c-1425-ec11-b6e6-000d3adf095a' =>
-          { name: 'HY Education Limited',
```
[the remark this answers is not in the document]
Comment (edwardhorsford) [reply lib/dfe/reference_data/degrees/institutions.rb:1935]: Done

File: lib/dfe/reference_data/degrees/institutions.rb (hunk 2 above)
Comment (edwardhorsford) [reply lib/dfe/reference_data/degrees/institutions.rb:2391]: It looks like both are used on the website - though possibly `UA92` a bit more commonly. My thinking here was that the abbreviation is mildly ambiguous (moreso for non UA92 people) so spelling it out might be more explicit. We don't have to make the change though.

**Pull request merged**